xwiki-platform-core/xwiki-platform-extension/xwiki-platform-extension-handlers/xwiki-platform-extension-handler-xar/src/main/resources/ApplicationResources.properties

```diff
@@ -48,11 +48,11 @@
 ###############################################################################
 
 extension.xar.log.importing=Importing document [{0}] in language [{1}]
-extension.xar.log.importing=Done installing document [{0}]
-extension.xar.log.importing=Failed to install document [{0}]
+extension.xar.log.importing=Document installed [{0}]
+extension.xar.log.importing=Document installation failed [{0}]
```

Comment:
The `[{0}]` is a place holder for the document name so it will looks a bit weird in your version.

Reply:
Sorry, I hadn't realised the function of placeholders here.